src/pkgcheck/checks/codingstyle.py

```diff
@@ -79,6 +79,64 @@ def feed(self, pkg):
                     yield DeprecatedEapiCommand(name, line=call, lineno=lineno+1, eapi=pkg.eapi, pkg=pkg)
 
 
+class EbeginMissingEend(results.LineResult, results.Style):
+    """Ebuild calls ebegin in a phase, but is missing a call to eend"""
+
+    def __init__(self, func_name, **kwargs):
+        super().__init__(**kwargs)
+        self.func_name = func_name
+
+    @property
+    def desc(self):
+        return f'Call to ebegin with a missing call to eend ({self.func_name}), on line {self.lineno}'
```

Comment:
To match the brevity and formatting of result descriptions this should be something like:

```python
return f'ebegin missing corresponding eend, line {self.lineno}'
```